pkg/apis/config/store.go

```diff
@@ -49,16 +49,12 @@ func FromContextOrDefaults(ctx context.Context) *Config {
 	if cfg := FromContext(ctx); cfg != nil {
 		return cfg
 	}
-	defaults, _ := NewDefaultsFromMap(map[string]string{})
-	featureFlags, _ := NewFeatureFlagsFromMap(map[string]string{})
-	metrics, _ := newMetricsFromMap(map[string]string{})
-	spireconfig, _ := NewSpireConfigFromMap(map[string]string{})
 
 	return &Config{
-		Defaults:     defaults,
-		FeatureFlags: featureFlags,
-		Metrics:      metrics,
-		SpireConfig:  spireconfig,
```

Comment:
this seems like an unrelated cleanup, is that right? can it go in a separate commit and pull request?

Reply:
Yes it's sort of unrelated, but really motivated by this pr - where we need to get the default configmap at many places.

Happy to separate it into another pr.

Reply:
please open another PR with the clean up and I'll prioritize reviewing it 🙏🏾

Reply:
SG https://github.com/tektoncd/pipeline/pull/6639. PTAL. Thanks!